test/integration/epacems_test.py

```diff
@@ -29,59 +25,29 @@ def epacems_parquet_path(pudl_engine: sa.Engine) -> Path:
     Args:
         pudl_engine: An implicit dependency that ensures the .parquet files exist.
     """
-    context = build_init_resource_context()
-    return (
-        epacems_io_manager(context)._base_path
-        / "core_epacems__hourly_emissions.parquet"
-    )
+    return PudlPaths().parquet_path("core_epacems__hourly_emissions")
 
 
-def test_epacems_subset(epacems_settings: EpaCemsSettings, epacems_parquet_path: Path):
-    """Check that epacems output retrieves a non-empty dataframe."""
+def test_epacems_output(epacems_settings: EpaCemsSettings, epacems_parquet_path: Path):
```

Comment:
Still testing whether we can read some data out of a Parquet file and get something reasonable.